generator/search_index_builder.cpp

```diff
@@ -281,10 +281,21 @@ void GetUKPostcodes(string const & filename, storage::CountryId const & countryI
     if (find(countries.begin(), countries.end(), countryId) == countries.end())
       continue;
 
+    // UK postcodes format are: aana naa, ana naa, an naa, ann naa, aan naa, aann naa.
```

Comment:
s/format/formats/ ?
is it one format or 6 different formats?

Reply:
done